metadata/expo.xml

```diff
@@ -7,12 +7,12 @@
 		<option name="toggle" type="activator">
 			<_short>Toggle</_short>
 			<_long>Shows an overview of all workspaces with the specified activator.  Pressing again exits.</_long>
-			<default>&lt;super&gt; KEY_E</default>
+			<default>&lt;super&gt;</default>
```

Comment:
I don't think this is a good choice for a default but maybe @ammen99 can chime in here.

Reply:
I use `<super>` for expo personally so +1 from me :P